Rmarkdown/Multiple_Ascending_Dose_PK.Rmd

```diff
@@ -1,573 +1,451 @@
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_annotate_filenames(dirs)
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width,height,dirs,"filename_main",status)
+print(gg)
+
+```
+
+## Assess the dose linearity of exposure
+
+### Dose Normalized Concentration over Time, colored by dose, mean +/- 95% CI
+
+```{r , cache = TRUE, echo = TRUE, warning=FALSE, message=FALSE, fig.width = 10, fig.height = 5}
+
+gg <- ggplot(data = pk_data, 
+             aes(x = NOMTIME, y = LIDV_NORM, 
+                 group= TRTACT_high2low, color = TRTACT_high2low)) 
+gg <- gg + xgx_theme()
+gg <- gg + xgx_geom_ci(conf_level = 0.95, alpha = 0.5)
+gg <- gg + xgx_scale_y_log10()
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+
+gg <- gg + labs(y = concnorm_label, color = trtact_label)
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_annotate_filenames(dirs)
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width,height,dirs,"filename_main",status)
+print(gg)
+ 
+```
+
+### Side-by-side comparison of first administered dose and steady state
+
+```{r , cache = TRUE, echo = TRUE, warning=FALSE, message=FALSE, fig.width = 10, fig.height = 5}
+
+gg %+% 
+  (pk_data %>% subset(PROFDAY %in% c(1,6))) %+% 
+  aes(x = PROFTIME) + 
+  xgx_scale_x_time_units(units_dataset = time_units_dataset,
+                         units_plot    = "Hours") + 
+  facet_grid(~DAY_label) + 
+  xlab("Time after dose (hours)") 
+
+```
+
+### NCA of dose normalized AUC vs Dose
+
+Observe the dose normalized AUC over different doses. Does the relationship appear to be constant across doses or do some doses stand out from the rest? Can you think of reasons why some would stand out? For example, the lowest dose may have dose normalized AUC much higher than the rest, could this be due to CENS observations? If the highest doses have dose normalized AUC much higher than the others, could this be due to nonlinear clearance, with clearance saturating at higher doses? If the highest doses have dose normalized AUC much lower than the others, could there be saturation of bioavailability, reaching the maximum absorbable dose?
+
+```{r, cache = TRUE, echo = TRUE, warning=FALSE, message=FALSE, fig.width = 8, fig.height = 4}
+
+gg <- ggplot(data = NCA %>% subset(PARAM %in% c("AUC_0_24", "AUC_tau")), aes(x = DOSE, y = VALUE_NORM))
+gg <- gg + geom_boxplot(aes(group = DOSE)) + geom_smooth(method = "lm", color = "black")
+gg <- gg + scale_x_continuous(breaks=unique(NCA$DOSE))
+gg <- gg + ylab("Dose normalized AUCtau (h.(ng/mL)/mg)") 
+gg <- gg + xlab("Dose (mg)")
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_annotate_filenames(dirs)
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width,height,dirs,"filename_main",status)
+print(gg + facet_grid(.~DAY_label))

```

Comment:
this is weird.  for consistency, i think it should be gg <- gg + facet_grid(...) further up.  And then print(gg) at the end.

Reply:
wanted to reuse gg without facet for the next plot